packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -276,20 +273,48 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
     );
   }
 
+  private loadingAnimationContent() {
+    const {isLoading, isStreaming} = this.generatedAnswerState;
+
+    if (isLoading || this.shouldBeHidden) {
+      return (
+        <div class="placeholder-wrapper relative overflow-hidden">
+          <GeneratedAnswerPlaceholder />
+        </div>
+      );
+    } else if (isStreaming) {
+      return (
+        <div class="placeholder-wrapper relative overflow-hidden">
+          <div class="z-1 bg-white relative">{this.renderContent()}</div>
+          <div class="absolute inset-0">
+            <GeneratedAnswerPlaceholder />
+          </div>
+          <div class="typing-loader-wrapper absolute inset-x-0 bottom-0 w-full z-1 pt-40">
+            <TypingLoader />
+          </div>
+        </div>
+      );
+    } else {
+      return this.renderContent();
+    }
+  }
+
   public render() {
     const {isLoading} = this.generatedAnswerState;
-    if (this.shouldBeHidden) {
-      return null;
-    }
+    isLoading
```

Comment:
This "synchronize" removing the result list placeholder (and thus rendering results) when we know there's an answer.

This means we can remove some initial flicker by making sure we display or hide skeletons for both genai + results at the same time.